typescript/node-js/client/src/app/users/usersWelcome.component.tsx

```diff
@@ -38,8 +28,11 @@ export default function UsersWelcome({ user }: { user: string | undefined }) {
           elevation={3}
           className='w-2/5 p-7 flex flex-col items-center gap-5'>
           <div>Welcome {user}!</div>
-          <Button href='/onboarding' variant='outlined'>
-            Start Employee Onboarding
+          <Button href='/onboarding?user-data=single' variant='outlined'>
+            Start Employee Onboarding {"(Single User Data)"}
+          </Button>
+          <Button href='/onboarding?user-data=multiple' variant='outlined'>
```

Comment:
Tambahi `className='text-center'` di button supaya kalau window browser kecil, tulisannya centered di tengah

Reply:
done https://github.com/gl-zkPass/zkpass-sdk/pull/79/commits/18b67539920064d0965b1a8551f357293850313e

Reply:
yang button `Back to home`, cuma muncul kalo proofnya success, kalau proof gagal ga muncul. Expectku muncul dua-duanya.

Terus mending Button nya outlined, kalau cuma text dia bisa ngira itu bukan button, cuma text.

Tulisannya ubah dari `Bach to home` jadi `Back to Home`

Reply:
done, salah taruh
https://github.com/gl-zkPass/zkpass-sdk/pull/79/commits/4643c8a9ce72783ae29eadbbaa882efd4e40d9e3
https://github.com/gl-zkPass/zkpass-sdk/pull/79/commits/532edeac476992a64ef9b88a06a908fa3f4eff59